### ISSUE TITLE
Is it normal that I can publish a build with the same version number as previous published extension ?

### ISSUE THREAD

**Issue by alexandredepape** [fregante/chrome-webstore-upload-cli#72]
Title :)

Thanks in advance ! :)

**Comment by fregante** [comment]
What do you mean? Probably you can upload a new zip with the same version over and over, but publishing the same version is not allowed by the store and therefore the API call should fail.

**Comment by alexandredepape** [comment]
Here is my yml:
```yml
name: Publish to Chrome Web Store

on:
  push:
    branches:
      - main

jobs:
  build:
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v2
      - uses: actions/setup-node@v2
        with:
          node-version: 18

      - name: Install dependencies
        run: yarn

      - name: Build with Yarn
        run: yarn run prod

      - name: Zip
        #run: zip -r chrome-extension-${{ github.event.workflow_run.head_sha }}.zip production
        run: zip -r build.zip production

      - name: Install webstore cli
        run: npm install -g chrome-webstore-upload-cli

      - name: Publish to Chrome Web Store
        run: chrome-webstore-upload publish --source build.zip --extension-id ${{ secrets.EXTENSION_ID }} --client-id ${{ secrets.CLIENT }} --client-secret ${{ secrets.SECRET }} --refresh-token ${{ secrets.TOKEN }}
```
<img width="1050" alt="Screenshot 2023-12-12 at 14 07 32" src="https://github.com/fregante/chrome-webstore-upload-cli/assets/25705704/bcda5b02-03bb-440f-b102-d98b59f8d74c">

I even get a email from the store: 

<img width="746" alt="Screenshot 2023-12-12 at 14 09 06" src="https://github.com/fregante/chrome-webstore-upload-cli/assets/25705704/36eed8c4-d085-4686-819f-a2b93106890f">

Multiple times, even thought the version number is the same. Because I have a bug at the same time with the modification of my version in my manifest.json: It is supposed to get changed but doesn't.

**Comment by fregante** [comment]
I see, I think you can actually submit the same version for review multiple times **but** not after the version has been approved. 

Either way it's not something that needs changing here.

**Comment by alexandredepape** [comment]
<img width="1226" alt="Screenshot 2023-12-12 at 14 42 40" src="https://github.com/fregante/chrome-webstore-upload-cli/assets/25705704/2e4a76b6-ec3a-4769-a89c-63e840d16319">

But it is published , look, published 1.30

**Comment by fregante** [comment]
You can report that on google's forums/groups
